css/styles.css

```diff
@@ -0,0 +1,105 @@
+@import "https://fonts.googleapis.com/css2?family=Asap:wght@100;300;400;500;600&display=swap";
+
+* {
+  box-sizing: border-box;
+  background-color: #f0f0f0;
+  font-family: Asap, sans-serif;
+}
+
+.header-title {
+  background-color: #552b80;
+  color: #f0f0f0;
+  padding: 1.2rem 7.2rem;
```

Comment:
You should avoid using larger `rem` values. If you're doing anything above `5rem`, you're probably doing something "hacky" to try and get this to work